LibraryApp/src/app/components/BookList/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { View, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+import Book from './components/Book';
+
+class BookList extends Component {
+  keyExtractor = ({ id }) => id;
+
+  renderItem = ({ item }) => <Book title={item.title} author={item.author} photo={item.image_url} />;
+
+  render() {
+    const { data } = this.props;
+    return (
+      <View style={styles.container}>
```

Comment:
Do you need this?

Reply:
i love views (? but really i don't need it 👍